src/conda_package_handling/conda_fmt.py

```diff
@@ -142,3 +142,10 @@ def get_pkg_details(in_file):
         size = stat_result.st_size
         md5, sha256 = utils.checksums(in_file, ("md5", "sha256"))
         return {"size": size, "md5": md5, "sha256": sha256}
+
+    @staticmethod
+    def list_contents(fn, verbose=False, **kw):
+        components = utils.ensure_list(kw.get("components")) or ("info", "pkg")
```

Comment:
I'm copying the same signature used in [other parts of the project](https://github.com/conda/conda-package-handling/blob/5668ac7eb671c5962841d7de5f110a60d2650a0a/src/conda_package_handling/conda_fmt.py#L39) for consistency. If we change it here it would be good to do it elsewhere as well.

Reply:
conda-package-streaming has a newer API, personally I would use the c-p-s API and only use c-p-h for backwards compatibility or its unique features. It also has [this enum for "info" and "pkg"](https://github.com/conda/conda-package-streaming/blob/main/conda_package_streaming/package_streaming.py#L30). If it's a .tar.bz2 you only use pkg and not both.